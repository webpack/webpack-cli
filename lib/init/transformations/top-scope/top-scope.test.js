"use strict";

const defineTest = require("../../../utils/defineTest");

defineTest(__dirname, "top-scope", "top-scope-0", ["const test = 'me';"], "init");
defineTest(
	__dirname,
	"top-scope",
	"top-scope-1",
	["const webpack = require('webpack');"],
	"add"
);
