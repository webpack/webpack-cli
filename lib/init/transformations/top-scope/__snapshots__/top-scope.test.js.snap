// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`top-scope transforms correctly using "top-scope-0" data 1`] = `
"const test = 'me';
module.exports = {}
"
`;

exports[`top-scope transforms correctly using "top-scope-1" data 1`] = `
"const webpack = require(\\"webpack\\");

module.exports = {}
"
`;
